prac_05/state_names.py

```diff
@@ -4,15 +4,19 @@
 File needs reformatting
 """
 
+state_dict = {"QLD": "Queensland", "NSW": "New South Wales",
```

Comment:
I believe the naming convention for dictionaries is keys_to_values, overall code looks well-written :)